x/consensus/keeper/consensus/consensus.go

```diff
@@ -342,6 +343,24 @@ func (c Queue) Remove(ctx context.Context, msgID uint64) error {
 		types.ItemRemovedEventID.With(fmt.Sprintf("%d", msgID)),
 		types.ItemRemovedChainReferenceID.With(c.qo.ChainReferenceID),
 	)
+
+	logger := liblog.FromSDKLogger(sdkCtx.Logger()).WithFields("msg_id", msgID)
```

Comment:
```suggestion
	logger := liblog.FromSDKLogger(sdkCtx.Logger()).WithFields("msg-id", msgID)
```

Logging fields are always [kebab-case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case)